src/infrastructure/routes/validation/support.inputs.ts

```diff
@@ -26,3 +26,17 @@ export class ChangerAgenceConseillerPayload {
   @IsString()
   idNouvelleAgence: string
 }
+
+export class TransfererJeunesPayload {
```

Comment:
je viens d'avoir une idée

vu que c'est le support qui fait ça, on devrait ne le laisser transferer qu'un jeune à la fois (et nous on traduit en tableau)

what do you think